### DISCUSSION TITLE
In the "books" example the ability to navigate (forward button) is lost when clicking the back button in the browser

### DISCUSSION THREAD

**Discussion by jamolina2021** [csells/go_router#365] [Q&A]
Hi,

In the "books" sample application, being in http://localhost:49752/#/books/popular, if I click on Authors, then on Settings, with the Back Button of the browser I can go back to Authors and to Books, at that moment, with the Forward Button, I can go back to Authors and to Settings , but if, being in books/popular url, I click the Back Button again, the browsing history is lost, and I can no longer go to Authors and Settings with the Forward Button. 
Please watch the following video: 
https://recordit.co/yumY1qir40
Is this a bug?

Thanks for the help. 

Regards,
Jose

**Comment by csells** [comment]
@jamolina2021 when I watch the video, the forward button seems to behave correctly. what am I missing?

**Comment by ghost** [comment]
@csells, 
No, as you can see in the video if you click on the back button you reach a point where the forward button is greyed out, so you can no longer go forward to the pages that are further ahead.
